libvector/libvector.h

```diff
@@ -19,7 +19,7 @@ typedef void *vector_iter;
 
 typedef struct vector {
     size_t size;     //number of element
-    size_t max_size; //max number of element
+    //size_t max_size; //max number of element
```

Comment:
no need comments "max_size" , [maximum number of elements that the vector can hold](http://www.cplusplus.com/reference/vector/vector/max_size/)